src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -2398,6 +2410,9 @@ private function registerNotifierConfiguration(array $config, ContainerBuilder $
             GitterTransportFactory::class => 'notifier.transport_factory.gitter',
             ClickatellTransportFactory::class => 'notifier.transport_factory.clickatell',
             MicrosoftTeamsTransportFactory::class => 'notifier.transport_factory.microsoftteams',
+            LightSmsTransportFactory::class => 'notifier.transport_factory.lightsms',
+            SmsBiurasTransportFactory::class => 'notifier.transport_factory.smsbiuras',
+            MessageBirdTransport::class => 'notifier.transport_factory.messagebird',
```

Comment:
Should be in a separate PR as bugfix on `5.x`, thanks

Reply:
https://github.com/symfony/symfony/pull/40842